src/misc/setup.cpp

```diff
@@ -988,7 +988,7 @@ std::string Section_line::GetPropValue(const std::string&) const
 	return NO_SUCH_PROPERTY;
 }
 
-bool Config::PrintConfig(const std::string& filename) const
+bool Config::WriteConfig(const std::string& filename) const
```

Comment:
I would rename `filename` to `path` since it wants a full path name.

Maybe it should be a `std_fs::path` too but it just gets passed to `fopen` so you could also just leave it as a string since it doesn't matter much in this use case.

Reply:
Good point, done.